SPECS/openssl/openssl.spec

```diff
@@ -9,7 +9,7 @@
 Summary: Utilities from the general purpose cryptography library with TLS implementation
 Name: openssl
 Version: 3.1.4
-Release: 1%{?dist}
+Release: 2%{?dist}
```

Comment:
Let's leave it at Release 1 for now -- we'll lock the release in later on, but this way we can work incrementally without resulting in a huge release number at ship date.